src/app/helptext/system/ntpservers.ts

```diff
@@ -38,17 +38,20 @@ export const helptext_system_ntpservers = {
     minpoll: {
       placeholder: T("Min. Poll"),
       tooltip: T(
-        "Power of 2 in seconds; cannot be lower than 4 or\
- higher than <i>Max. Poll</i>."
+        "Calculated in powers of 2. Enter the exponent to be \
+ calculated. For example, if the <i>Min. Poll</i> is 6, the \
+ calculation is 2^6. Cannot be lower than 4 or higher than \
+ <i>Max. Poll</i>."
       ),
       validation: [Validators.min(4), Validators.required]
     },
 
     maxpoll: {
       placeholder: T("Max. Poll"),
       tooltip: T(
-        "Power of 2 in seconds; cannot be higher than 17 or\
- lower than <i>Min. Poll</i>."
+        "Calculated in powers of 2. Enter the exponent to be \
```

Comment:
`The maximum polling interval, in seconds, as a power of 2. For example, <i>10</i> means 2^10, or 1,024 seconds. The default is 10, maximum value is 17.`